local-cli/server/server.js

```diff
@@ -92,6 +92,11 @@ module.exports = {
     parse: (val) => val.split(','),
     default: (config) => config.getProjectRoots(),
   }, {
+    command: '--sourceExts [list]',
+    description: 'Specify any source extentions to be used by the packager, defaults to js',
```

Comment:
Maybe this can be made slightly more helpful by mentioning that this can be used in combination with `--transformer` to process other file types?